genesyscloud/resource_genesyscloud_journey_segment_test.go

```diff
@@ -106,36 +107,3 @@ func testVerifyJourneySegmentsDestroyed(state *terraform.State) error {
 	// Success. All Journey segment destroyed
 	return nil
 }
-
-func customerSegmentationIsSupported(t *testing.T) (bool, string) {
```

Comment:
This was used before the previous PR to skip some tests, is no longer being used at all, so I'm removing it.